components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -1571,6 +1573,35 @@ func trackReadiness(ctx context.Context, gitpodService *gitpod.APIoverJSONRPC, c
 	}
 }
 
+func configureSSHDefaultDir() {
+	file, err := os.OpenFile("/home/gitpod/.bashrc", os.O_WRONLY|os.O_APPEND, 0o644)
+	if err != nil {
+		log.WithError(err).Error("cannot write .bashrc")
+	}
+	defer file.Close()
+	if _, err := file.WriteString("\nif [[ -n $SSH_CONNECTION ]]; then cd \"$GITPOD_REPO_ROOT\"; fi\n"); err != nil {
+		log.WithError(err).Error("write .bashrc failed")
+	}
+}
+
+func configureSSHMessageOfTheDay() {
+	msg := []byte(`Welcome to Gitpod - Always ready to code. You can use commands below to be more productive.
```

Comment:
I can't find a proper char to scroll terminal content to next page without clear history, (tried chars in [this page](https://www2.ccs.neu.edu/research/gpc/VonaUtils/vona/terminal/vtansi.htm) on my computer)